CMakeLists.txt

```diff
@@ -89,10 +93,12 @@ target_link_libraries(threaded_io_test
                               ROOT::RIO
                               ROOT::Tree
                               TBB::tbb
+                              stdc++
                               Threads::Threads
                               configKeys
                               sequence_classes_dict
                               zstd::libzstd_shared
+                              hdf5_hl
```

Comment:
Does this come from the hdf5 cmake package?

Reply:
No. hdf5_hl.h is a separate include file, so does the static/dynamic library. It can be disabled at compile time for HDF5. One of my HDF5 strategy depends on this, so we need to explicitly bring it here.